src/lib/storage/index/base_index.hpp

```diff
@@ -38,6 +38,19 @@ class BaseIndex : private Noncopyable {
   // For now we use an iterator over a vector of chunkoffsets as the GroupKeyIndex works like this
   using Iterator = std::vector<ChunkOffset>::const_iterator;
 
+  /**
+   * Predicts the memory consumption in bytes of creating an index with the specific index implementation <type>
+   * on a Chunk with the following statistics:
+   *
+   * row_count - overall number of rows
+   * value_count - number of distinct values
```

Comment:
For example, in the histograms we use `distinct_count` and probably at other places throughout the code. Should we stick with that?

Reply:
For me, `value_count` is not the same as a `distinct_count`.

Reply:
changed to `distinct_count`.